### PR TITLE
First code copied from Skoltech repo

### PR DESCRIPTION
**Pull request by dastels**

### CODE REVIEW

**Comment by ladyada** [comment]
looks great so far - let me know when you want 👁 review/suggestions!

**Comment by dastels** [comment]
@ladyada This works fine opn the Pi with CPython using the USB interface.  But reading from the UART under CP seems to be returning random bytes.

**Comment by ladyada** [comment]
huh thats odd, well we will have to figure that part out when we use it on an M4

**Comment by dastels** [comment]
Yeah, that on an M4.

**Comment by dastels** [comment]
@kattni When you have a chance.

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
-# Adafruit_CircuitPython_RPLIDAR
\ No newline at end of file
+## Introduction ##
```
Comment (kattni) [inline README.md]: You shouldn't have two READMEs. The README.rst is the one rendered by RTD, and is the one we use. Please transfer any information you wish to keep from this README to the README.rst file and delete the README.md file.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import setuptools
```
Comment (kattni) [inline setup.py]: Where did you get this setup.py? Do you intend to have this on PyPI through Travis the way we have done with every other library? If so, this is not a sufficient file.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# This is a common .travis.yml for generating library release zip files for
```
Comment (kattni) [inline .travis.yml]: This is not the most recent version of `.travis.yml`. Please update it.

**Comment by TheKitty** [comment]
@dastels - please correct any link to the code in the guide.

**Pull request merged**